logger/src/lib.rs

```diff
@@ -121,12 +120,15 @@ pub fn disable_global() -> Result<()> {
     try_set_logger()
 }
 
+#[allow(clippy::needless_pass_by_value)]
 fn step2<L>(config: InitConfig, layer: L) -> Result<LoggerHandle>
 where
     L: tracing_subscriber::Layer<Registry> + Debug + Send + Sync + 'static,
 {
-    let level: tracing::Level = into_tracing_level(config.base.level);
-    let level_filter = tracing_subscriber::filter::LevelFilter::from_level(level);
+    // NOTE: unfortunately constructing EnvFilter from vector of Directive is not part of public api
```

Comment:
There's a [PR open](https://github.com/tokio-rs/tracing/pull/2763), not sure why it wasn't merged

Reply:
Let's hope it will be merged soon and we would have more ergonomic API